### ISSUE TITLE
exception:  suscan_config_set_float() , suscan_config_desc_lookup_field_id()

### ISSUE THREAD

**Issue by jedi7** [BatchDrake/SigDigger#249]
Hi,
I'm using following (actual) commits:
* sigutils: 0e5b64a763d40c111ec5346985713ca6c0fbef32
* suscan: d21f18cfa5ba53c0bfea7032b0b527df65dcfaa8
* suwidgets: f7571f74ef064ad0ee516e088ae8183ba3f36d78
* sigdigger: 60a9d38cd5f0ac79d5fb1df7757b9dc31e1baf4a

folder .suscan was deleted before test

SDR is connected through soapysdr (remote)

Steps to reproduce:
* autodetect remote sdr
* Enable Baseband audio preview
* Start 

Then I get:
`C++ API exception: expression `suscan_config_set_float(this->instance, name.c_str(), val)' failed in include/Suscan/Config.h:115
info: Source does not support native DC correction, falling back to software correction
info: DC correction strategy: continuous
error: exception in "(id = suscan_config_desc_lookup_field_id(cfg->desc, name)) != -1" (util/cfg.c:455)`

**Comment by BatchDrake** [comment]
We are merging all projects to master and abandon develop as development
branch. Right now the transition is complete in sigutils and Suscan, but
not in SuWidgets and SigDigger.

TL;DR build sigutils and Suscan from master, SuWidgets and SigDigger from
develop, until further notice.
